src/Sulu/Component/Content/Document/Subscriber/WebspaceSubscriber.php

```diff
@@ -64,11 +63,7 @@ public function doHydrate(AbstractMappingEvent $event)
         $event->getAccessor()->set('webspaceName', $webspaceName);
     }
 
-    /**
-     * @param PersistEvent $event
-     */
     public function doPersist(PersistEvent $event)
```

Comment:
Shouldn't this method be removed if we don't serve the `Events::PERSIST` event anymore?